public/app/features/integrated-alerting/components/AlertRuleTemplate/EditAlertRuleTemplateModal/EditAlertRuleTemplateModal.messages.ts

```diff
@@ -1,7 +1,8 @@
 export const Messages = {
   alertRuleTemplateLabel: 'Alert Rule Template',
   submitButton: 'Save',
-  title: 'Edit Alert Rule Template',
   cancelAction: 'Cancel',
   editSuccess: 'Alert rule template successfully edited',
+  nameNotEditable: "Name can't be changed. If you need to change it, please create a new Template.",
```

Comment:
I would use "can not" or "cannot", since "can't" seem too informal. But I'd check with Paul first

Reply:
👍